src/addScore.js

```diff
@@ -0,0 +1,37 @@
+import {
+  form, user, score, apiURL,
+} from './variable.js';
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  async function addData() {
+    const add = await fetch(apiURL, {
```

Comment:
- [ ] It would be great if you use the arrow function **`const addData = async () => {}`** instead of the 
**`async function addData()`** to follow your project requirements :+1: